config/nonprod.tfvars

```diff
@@ -0,0 +1,13 @@
+env = "nonprod"
```

Comment:
this should be passed in by the terraform not set via tfvars

Reply:
done